demo.tt

```diff
@@ -23,7 +23,7 @@ stemming algorithm:
 [% END %]
 <script>
 var old_words;
-function stem_demo(elt) {
+function (elt) {
```

Comment:
Won't this break the demo?

Reply:
Mmmmh I wasn't councious I changed that, but I tested it with the proposed function name and still says error...